doc/source/admin/special_topics/performance_tracking.rst

```diff
@@ -5,34 +5,29 @@ Tracking performance over time and identifying slow queries in your Galaxy can b
 
 Most performance tracking requires sending metrics to a metrics collection server such as `Graphite <http://graphiteapp.org/>`__ or `StatsD <https://github.com/etsy/statsd/>`__. This document assumes you have already deployed a metrics server.
 
-uWSGI
+Gunicorn
 -----
 
-There is some built-in uWSGI support for performance logging. You can send uWSGI's internal metrics to a carbon (Graphite) server by setting the carbon option in your galaxy.yml:
+There is some built-in Gunicorn support for performance logging. You can send Gunicorn's internal metrics to a StatsD server by setting the `--statsd-host` and `--statsd-prefix` command line options for Gunicorn in the `gravity` section of `galaxy.yml`:
 
 .. code-block:: yaml
 
-   uwsgi:
-      socket: ...
-      carbon: 127.0.0.1:2003
+   gravity:
+      ...
+      gunicorn:
+        gunicorn_extra_args: `--statsd-host 127.0.0.1:8125 --statsd-prefix=galaxy`
```

Comment:
```suggestion
        gunicorn_extra_args: `--statsd-host 127.0.0.1:8125 --statsd-prefix=gunicorn`
```

I misunderstood the conversation here. Prefixing with `galaxy` would result in mixing gunicorn metrics with native galaxy metrics. This is exactly analogous to the carbon/statsd-push settings you're removing, so, we should 

I think prefixing here with gunicorn would make the most sense, since it's a separate component. (Or galaxy.gunicorn but that feels wrong to me?)

Reply:
Awesome, thanks!